x-pack/plugins/observability_solution/observability_ai_assistant/server/service/index.ts

```diff
@@ -122,7 +124,17 @@ export class ObservabilityAIAssistantService {
     });
   }
 
-  init = once(async () => {
+  init = async () => {};
+
+  private allowInit = () => {
+    this.init = once(async () => {
+      return this.doInit().catch(() => {
```

Comment:
can you rethrow here?

Reply:
Fixed in https://github.com/elastic/kibana/pull/179346/commits/ab09231b719470f6d975b7716a32659b5077f7f2

Good **_catch_**!